src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerProfile.java

```diff
@@ -157,12 +130,6 @@ public boolean isDirty() {
      * This method will save the Player's Researches and Backpacks to the hard drive
      */
     public void save() {
-        for (PlayerBackpack backpack : backpacks.values()) {
-            backpack.save();
-        }
-
-        waypointsFile.save();
-        configFile.save();
         dirty = false;
     }
```

Comment:
Shouldn't we call Storage#savePlayerData here before marking dirty as false?

Reply:
🤦 yes, ty